content/200-concepts/100-components/04-introspection.mdx

```diff
@@ -204,12 +204,12 @@ Prisma translates this into the following data model:
 
 ```prisma
 model User {
-  id      Int      @default(autoincrement()) @id
+  id      Int      @id @default(autoincrement())
```

Comment:
This is what the formatter does to the schema?

Reply:
@janpio I didn't use the formatter but it says [here](https://www.prisma.io/docs/concepts/components/introspection#order-of-attributes) that `@id` must come before `@default`. I also used `prisma introspect` in my own project and it did place `@id` before `@default`.

Reply:
Then you effectively used the same codebase as the formatter 👍